app/views/project_request_mailer/project_request.text.erb

```diff
@@ -2,7 +2,7 @@ Hi <%= @owner %>
 
 Thanks for requesting your project, <%= @name %>: <%= @url %>, be reviewed for <%= @type %>.
 
-Someone on our team will review your project and be in touch with you in the next few weeks about the next steps.
+Someone on our team will review your project and be in touch with you in the next few weeks about the next steps. To learn more about the project review process, timeline, and best practices, please visit <%= link_to "this link", "https://docs.google.com/document/d/1SJmOdGmpzYGyKpSnFt_tEe_BZIc2Bzmc3kKlWw-gX68/edit?usp=sharing" %>.
```

Comment:
I'm not sure you should use the link_to helpers here as it generates html, ttps://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to

You already have the full URL and this is a text template so just use the text :)
```suggestion
Someone on our team will review your project and be in touch with you in the next few weeks about the next steps. Read our project review process, timeline, and best practices document to learn more https://docs.google.com/document/d/1SJmOdGmpzYGyKpSnFt_tEe_BZIc2Bzmc3kKlWw-gX68.
```

Reply:
Ah -- I now understand the `.text.erb` template a bit better.  None of the variables will produce clickable links in the body text (but providers like Gmail will typically convert these anyway), so just spell out the full URL.  Got it.  Thanks!